src/core/reference/include/openvino/reference/scatter_nd_update.hpp

```diff
@@ -8,18 +8,92 @@
 #include <numeric>
 
 #include "openvino/core/shape.hpp"
+#include "openvino/op/scatter_nd_update.hpp"
+#include "openvino/reference/maximum.hpp"
+#include "openvino/reference/minimum.hpp"
+#include "openvino/reference/multiply.hpp"
+#include "openvino/reference/xor.hpp"
+#include "subtract.hpp"
```

Comment:
```suggestion
#include "openvino/reference/subtract.hpp"
```

Reply:
Thanks, missed this include, removed instead